index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="web " content="width=device-width, initial-scale=1.0">
+    <title>Calculator</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>ENTER TWO NUMBERS TO ADD TOGETHER</h1>
+
+  
+    <input class="box" type="number" name="firstBox" id="number">
+    <br />
+
+
+    <input class="box" type="number" name="secondBox" id="number1">
+
+    <br />
+
+    <input class="box" type="number" name="thirdBox" id="result">
+
+
+    <input class="click" type="button" name="click"  value="AddNumber"  onclick="calculate()">
+  
+  
+        
+<div class="pattern">
+    <div class="circle1"></div>
+    <div class="circle2"></div>
+    <div class="circle3"></div>
+    <div class="circle4"></div>
+    <div class="circle5"></div>
+</div>
+    
+</body>
+<script type="text/javascript" src="script.js"></script>
```

Comment:
Only add your script after your html if you are optimizing for load time. In this case, such a small script can go in your head block.